clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -578,21 +587,25 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
 
     if (memref.exit.type == TRACE_TYPE_THREAD_EXIT) {
         shard->saw_thread_exit_ = true;
-        report_if_false(shard,
-                        !TESTANY(OFFLINE_FILE_TYPE_FILTERED | OFFLINE_FILE_TYPE_IFILTERED,
-                                 shard->file_type_) ||
-                            shard->found_instr_count_marker_,
-                        "Missing instr count markers");
-        report_if_false(shard,
-                        shard->found_cache_line_size_marker_ ||
-                            (shard->skipped_instrs_ && !is_a_unit_test(shard) &&
-                             shard->stream->get_cache_line_size() > 0),
-                        "Missing cache line marker");
-        report_if_false(shard,
-                        shard->found_page_size_marker_ ||
-                            (shard->skipped_instrs_ && !is_a_unit_test(shard) &&
-                             shard->stream->get_page_size() > 0),
-                        "Missing page size marker");
+        if (!TESTANY(OFFLINE_FILE_TYPE_KERNEL_SYSCALL_TRACE_TEMPLATES,
+                     shard->file_type_)) {
+            report_if_false(
+                shard,
+                !TESTANY(OFFLINE_FILE_TYPE_FILTERED | OFFLINE_FILE_TYPE_IFILTERED,
+                         shard->file_type_) ||
+                    shard->found_instr_count_marker_,
+                "Missing instr count markers");
```

Comment:
Wait, why does this need to be disabled?  Are syscall templates marked filtered??  But they're not filtered.